build.sh

```diff
@@ -110,7 +110,7 @@ fi
 # Build fatbins
 SRC="cpp/src"
 FAT="python/cusignal"
-FLAGS="-std=c++11"
+FLAGS="-std=c++17"
```

Comment:
have you seen any issues with setting the c++17 flag?

Reply:
No issues for our stuff.
I'm using `if constexpr` to minimize runtime branching when selecting device based on type. It requires C++17.